File: sample-chat/src/main/java/com/quickblox/sample/chat/utils/qb/callback/QbEntityCallbackImpl.java
package com.quickblox.sample.chat.utils.qb.callback;

import android.os.Bundle;

import com.quickblox.core.QBEntityCallback;
import com.quickblox.core.exception.QBResponseException;

public class QbEntityCallbackImpl<T> implements QBEntityCallback<T> {

    public QbEntityCallbackImpl() {
    }

    @Override
    public void onSuccess(T result, Bundle bundle) {

    }

    @Override
    public void onError(QBResponseException e) {

    }
}

File: sample-chat-kotlin/app/src/main/java/com/quickblox/sample/chat/kotlin/utils/qb/callback/QbEntityCallbackImpl.kt
package com.quickblox.sample.chat.kotlin.utils.qb.callback

import android.os.Bundle
import com.quickblox.core.QBEntityCallback
import com.quickblox.core.exception.QBResponseException


open class QbEntityCallbackImpl<T> : QBEntityCallback<T> {

    override fun onSuccess(result: T, bundle: Bundle?) {

    }

    override fun onError(e: QBResponseException) {

    }
}